installation/manifests/nightly/000-subscription-amq-streams.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
why are these in `nightly` ? they seem to be more static ...

Reply:
`nightly` or `product` under `installation/manifests` represents the system under test

Reply:
we're using AMQ streams for these tests but we don't want to couple the Kafka provider to a test since we might need to use a different Kafka provider.